cypress/integration/admin/account/account.spec.js

```diff
@@ -26,4 +26,28 @@ context("Account Module", () => {
             cy.should("exist");
         });
     });
+
+    it("should be able to change avatar", () => {
+        cy.visit("/account");
+        cy.get('div[data-role="select-image"]').click();
+
+        cy.findByTestId("fm-list-wrapper").dropFile("sample.jpeg", "image/jpeg");
+        cy.findByText("File upload complete.").should("exist");
+        cy.get('div[data-testid="fm-list-wrapper-file"]:first-child').click();
+
+        cy.get('div.mdc-layout-grid div > img').should('exist');
+       
+        cy.get('span').contains('Update account').click();
+        cy.contains('Account saved successfully').should('be.visible');
+
+        // Refresh page to assert the avatar persists.
+        cy.reload();
+        cy.get('div.mdc-layout-grid div > img').should('exist');
+
+        // Delete the newly avatar image.
+        cy.get('[data-testid="remove-image"]').click({force: true})
```

Comment:
We can use `findByTestId` selector for these.
P.S. I forgot to comment on this in our previous PR.